contracts/chain-adapters/ZkStack_Adapter.sol

```diff
@@ -158,8 +172,28 @@ contract ZkStack_Adapter is AdapterInterface {
                     refundRecipient: L2_REFUND_ADDRESS
                 })
             );
+        } else if (l1Token == address(usdcToken)) {
```

Comment:
@bmzig So for zkSync we'd require `usdcToken` to be 0x0, right? Then it's handled as a vanilla bridged ERC20.

Reply:
I'd think we want USDC to always be defined. This branch then will either bridge USDC via cctp or the canonical bridge.

Reply:
Oh sorry, I misunderstood your point. I was thinking we set `USDC_SHARED_BRIDGE` to the same thing as `SHARED_BRIDGE`, but your way works too.